meinberlin/apps/votes/models.py

```diff
@@ -60,12 +59,17 @@ def get_or_create_salt_for_module(module):
 
 class TokenPackage(models.Model):
     module = models.ForeignKey(Module, on_delete=models.CASCADE)
-    package_number = models.PositiveIntegerField()
     size = models.PositiveIntegerField()
     downloaded = models.BooleanField(default=False)
 
     class Meta:
-        unique_together = ["module", "package_number"]
+        ordering = ["pk"]
+
+    def get_num_tokens(self):
+        return VotingToken.objects.filter(package=self).count()
+
+    def created(self):
```

Comment:
`is_created` as that is what we call bools elsewhere. Like `is_draft`. Ok, we don't always do that. But `created` is still already taken in the TimeStampedModel as a time field. So, I would prefer `is_created`.
And you could add that as a property. Or maybe not. Does that result in that being done every DB call then? Have to read up on that.

Reply:
did add the property, also to num_tokens now